.storybook/manager-head.html

```diff
@@ -5,6 +5,9 @@
 <meta property="og:description" content="NeoPOP Design System by CRED" />
 <meta property="og:title" content="CRED Design System" />
 <title>NeoPOP Design System</title>
+<script>
```

Comment:
TBH this seems like a hack let's wait for storybook to fix this can we? It is already merged and made it to the pre release (https://github.com/storybookjs/storybook/releases/tag/v7.0.0-alpha.10) and then we can update the dependency instead of doing this? 

- https://github.com/storybookjs/storybook/pull/17935